src/cpptools.ts

```diff
@@ -22,13 +22,12 @@ const localize: nls.LocalizeFunc = nls.loadMessageBundle();
 const log = createLogger('cpptools');
 
 type Architecture = 'x86' | 'x64' | 'arm' | 'arm64' | undefined;
-type StandardVersion = "c89" | "c99" | "c11" | "c17" | "c++98" | "c++03" | "c++11" | "c++14" | "c++17" | "c++20"
-  | "gnu89" | "gnu99" | "gnu11" | "gnu17" | "gnu++98" | "gnu++03" | "gnu++11" | "gnu++14" | "gnu++17" | "gnu++20";
+type StandardVersion = "c89" | "c99" | "c11" | "c17" | "c++98" | "c++03" | "c++11" | "c++14" | "c++17" | "c++20" | "gnu89" | "gnu99" | "gnu11" | "gnu17" | "gnu++98" | "gnu++03" | "gnu++11" | "gnu++14" | "gnu++17" | "gnu++20";
```

Comment:
I think the C++ and C versions were split onto two lines to make it easier to find where they start and end.  Can you change this back?

Reply:
I copied this line from cpptools API (SourceFileConfiguration interface in api.d.ts) to make sure we are not missing any item. My intention was not to remove the new line. I can keep it like this to be similar to cpptools API, or split it between C and C++.

Reply:
I would like to keep the c++ and c versions on different lines.  [In api.ts of the cpptools API](https://github.com/microsoft/vscode-cpptools-api/blob/8ae425a623649b3f26bebad8e0917f11848506e8/api.ts#L176), you will see that they are split this way.  When api.d.ts is generated, the generator removes the newline.